conformance/conformance_test.go

```diff
@@ -42,26 +44,33 @@ func TestConformance(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Error initializing Kubernetes client: %v", err)
 	}
+	clientset, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		t.Fatalf("Error initializing Kubernetes REST client: %v", err)
+	}
+
 	v1alpha2.AddToScheme(client.Scheme())
 	v1beta1.AddToScheme(client.Scheme())
 
 	supportedFeatures := parseSupportedFeatures(*flags.SupportedFeatures)
 	exemptFeatures := parseSupportedFeatures(*flags.ExemptFeatures)
-	for feature := range exemptFeatures {
-		supportedFeatures.Delete(feature)
-	}
 
 	t.Logf("Running conformance tests with %s GatewayClass\n cleanup: %t\n debug: %t\n enable all features: %t \n supported features: [%v]\n exempt features: [%v]",
 		*flags.GatewayClassName, *flags.CleanupBaseResources, *flags.ShowDebug, *flags.EnableAllSupportedFeatures, *flags.SupportedFeatures, *flags.ExemptFeatures)
 
 	cSuite := suite.New(suite.Options{
 		Client:                     client,
+		RESTClient:                 clientset.CoreV1().RESTClient().(*rest.RESTClient),
```

Comment:
It looked like a simple omission here, given that the tests are known in the suite, they just failed when they were run because of the uninitialized RESTClient field. I can move it into a separate PR if you feel strongly, or update the title/description to mention this PR also initializes the RESTClient. Let me know what you'd prefer.

Reply:
eh I think its fair to just amend the description/resulting commit message, small enough change IMO (and unlikely to need to be reverted etc. later so ok combining with another)